cookbook/service_container/event_listener.rst

```diff
@@ -1,27 +1,36 @@
 .. index::
    single: Events; Create listener
+   single: Create subscriber
 
-How to Create an Event Listener
-===============================
+How to Create Event Listeners and Subscribers
+=============================================
 
-Symfony has various events and hooks that can be used to trigger custom
-behavior in your application. Those events are thrown by the HttpKernel
-component and can be viewed in the :class:`Symfony\\Component\\HttpKernel\\KernelEvents` class.
+Symfony has various events and hooks that can be used to perform custom
+actions in your application. Those events are triggered by the HttpKernel
+component and they are defined in the :class:`Symfony\\Component\\HttpKernel\\KernelEvents`
+class.
```

Comment:
Imo this paragraph contradicts a bit your other PR where you try to clarify that user's don't have to provider their own event dispatcher for custom events.